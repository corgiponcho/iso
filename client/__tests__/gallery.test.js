import React from 'react';
import Gallery from '../src/components/gallery/gallery';
import renderer from 'react-test-renderer';

test('Gallery component renders the text inside it', () => {
  const component = renderer.create(
    <div className="gallery-container">
      Gallery
    </div>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});