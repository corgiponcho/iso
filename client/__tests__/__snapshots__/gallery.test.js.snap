// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery component renders the text inside it 1`] = `
<div
  className="gallery-container"
>
  Gallery
</div>
`;
